libr/magic/d/default/extras

```diff
@@ -0,0 +1,79 @@
+#
+# Additional forensics magic file, suggested by #10554
+# This database is sourced from https://en.wikipedia.org/wiki/List_of_file_signatures
+#
+# Rules in making this database:
+# 1. based on file(1) magic for OpenBSD objects (NetBSD may have different ones)
+# 2. potential false-positive data is ruled out, means offset 0x0 less that 4bytes are out
+# 3. unknown endians are formed as hex strings
+# 4. tested fails, unsures, lack of reference ones are ruled out
+# 5. tested one were recorded, arch for tests: MIPS, Intel, ARM, OS: Linux,FreeBSD, OSX
+# 6. endianness were splitted - due to confirmed difference possibility on related systems
+#
+# This is DB in stage1 - planned to be added/corrected by stages upon having free time to do so.
+# blaim - unixfreaxjp for incorrect approach -Tue Sep 18 05:58:29 JST 2018
+#
+
+0	string	\x42\x41\x43\x4B\x4D\x49\x4B\x45\x44\x49\x53\x4B	AmiBack backup file
+0	string 	\x05\x07\x00\x00\x42\x4F\x42\x4F			AppleWorks document 
+0	string 	\x78\x01\x73\x0D\x62\x62\x60				Apple Disk Image file
+0	string 	\x41\x54\x26\x54\x46\x4F\x52\x4D			DjVu document
+0	string	\x27\x05\x19\x56					Universal Boot Loader file
+0	string	\x65\x87\x78\x56					PhotoCap Object Templates
+0	string	\x55\x55\xaa\xaa					PhotoCap Vector
+0	belong 	0xa1b2c3d4	Libpcap File Format, big endian
+0	lelong 	0xa1b2c3d4	Libpcap File Format, little endian
+0	belong 	0x0a0d0d0a	PCAPNG  File Format, big endian
+0	lelong 	0x0a0d0d0a	PCAPNG  File Format, little endian
+0	belong	0xedabeedb	RedHat RPM package
+0	lelong	0xedabeedb	RedHat RPM package
+0	belong	0x53503031	Amazon Kindle Update Package
+0	lelong	0x53503031	Amazon Kindle Update Package
+0	belong	0xbebafeca	Palm Desktop Calendar Archive 
+0	lelong	0xbebafeca	Palm Desktop Calendar Archive 
+0	belong	0x00014244	Palm Desktop To Do Archive 
+0	lelong	0x00014244	Palm Desktop To Do Archive 
```

Comment:
> All those palm* may be removed palm is dead

No problem, was thinking the same on the Palm is dead,.

> because will spot lot of false positives and 

I was suspecting a lot of false positive actually, but it was not during the check, maybe the sample range for tests need to be improved wider too. I don't mind to remove these.